backend/shared/cosmo_db.py

```diff
@@ -669,7 +669,7 @@ def get_company_list():
     try:
         items = list(
             container.query_items(
-                query="SELECT c.id, c.name, c.ticker, c.is_active, c.created_at, c.lastRun FROM c",
+                query="SELECT * FROM c",
```

Comment:
@vmlcode is all the data from companies required ? is this change need for this task?
is more efficient is we explicitly set the attributes instead of *